diff --git a/src/Service/RegexSpamWordHelper.php b/src/Service/RegexSpamWordHelper.php
index 9974497..7e2456a 100644
--- a/src/Service/RegexSpamWordHelper.php
+++ b/src/Service/RegexSpamWordHelper.php
@@ -2,7 +2,9 @@
 
 namespace App\Service;
 
-class RegexSpamWordHelper
+use App\Comment\CommentSpamCounterInterface;
+
+class RegexSpamWordHelper implements CommentSpamCounterInterface
 {
     public function getMatchedSpamWords(string $content): array
     {
