diff --git a/src/Service/RegexSpamWordCounter.php b/src/Service/RegexSpamWordCounter.php
index ffc05ed..77e9077 100644
--- a/src/Service/RegexSpamWordCounter.php
+++ b/src/Service/RegexSpamWordCounter.php
@@ -6,7 +6,13 @@ class RegexSpamWordCounter
 {
     public function countSpamWords(string $content): int
     {
+        $badWordsOnComment = [];
 
+        $regex = implode('|', $this->spamWords());
+
+        preg_match("/$regex/", $content, $badWordsOnComment);
+
+        return count($badWordsOnComment);
     }
 
     private function spamWords(): array
