diff --git a/src/Service/RegexSpamWordCounter.php b/src/Service/RegexSpamWordCounter.php
index 1c13d70..ffc05ed 100644
--- a/src/Service/RegexSpamWordCounter.php
+++ b/src/Service/RegexSpamWordCounter.php
@@ -4,5 +4,19 @@ namespace App\Service;
 
 class RegexSpamWordCounter
 {
+    public function countSpamWords(string $content): int
+    {
 
+    }
+
+    private function spamWords(): array
+    {
+        return [
+            'follow me',
+            'twitter',
+            'facebook',
+            'earn money',
+            'SymfonyCasts',
+        ];
+    }
 }
