diff --git a/src/Twig/AppExtension.php b/src/Twig/AppExtension.php
index 7be2947..4a18185 100644
--- a/src/Twig/AppExtension.php
+++ b/src/Twig/AppExtension.php
@@ -12,8 +12,6 @@ class AppExtension extends AbstractExtension
 {
     private $commentHelper;
 
-    private $userStatuses = [];
-
     public function __construct(CommentHelper $commentHelper)
     {
         $this->commentHelper = $commentHelper;
@@ -28,11 +26,7 @@ class AppExtension extends AbstractExtension
 
     public function getUserActivityText(User $user): string
     {
-        if (!isset($this->userStatuses[$user->getId()])) {
-            $this->userStatuses[$user->getId()] = $this->calculateUserActivityText($user);
-        }
-
-        return $this->userStatuses[$user->getId()];
+        return $this->calculateUserActivityText($user);
     }
 
     private function calculateUserActivityText(User $user): string
