diff --git a/src/Twig/AppExtension.php b/src/Twig/AppExtension.php
index 2cd5e63..7be2947 100644
--- a/src/Twig/AppExtension.php
+++ b/src/Twig/AppExtension.php
@@ -12,6 +12,8 @@ class AppExtension extends AbstractExtension
 {
     private $commentHelper;
 
+    private $userStatuses = [];
+
     public function __construct(CommentHelper $commentHelper)
     {
         $this->commentHelper = $commentHelper;
@@ -25,6 +27,15 @@ class AppExtension extends AbstractExtension
     }
 
     public function getUserActivityText(User $user): string
+    {
+        if (!isset($this->userStatuses[$user->getId()])) {
+            $this->userStatuses[$user->getId()] = $this->calculateUserActivityText($user);
+        }
+
+        return $this->userStatuses[$user->getId()];
+    }
+
+    private function calculateUserActivityText(User $user): string
     {
         $commentCount = $this->commentHelper->countRecentCommentsForUser($user);
 
