diff --git a/src/Service/SightingScorer.php b/src/Service/SightingScorer.php
index fb1c91b..99cedb1 100644
--- a/src/Service/SightingScorer.php
+++ b/src/Service/SightingScorer.php
@@ -4,6 +4,7 @@ namespace App\Service;
 
 use App\Entity\BigFootSighting;
 use App\Model\BigFootSightingScore;
+use App\Scoring\ScoreAdjusterInterface;
 use App\Scoring\ScoringFactorInterface;
 
 class SightingScorer
@@ -13,9 +14,15 @@ class SightingScorer
      */
     private iterable $scoringFactors;
 
-    public function __construct(iterable $scoringFactors)
+    /**
+     * @var ScoreAdjusterInterface[]
+     */
+    private $scoreAdjusters;
+
+    public function __construct(iterable $scoringFactors, iterable $scoreAdjusters)
     {
         $this->scoringFactors = $scoringFactors;
+        $this->scoreAdjusters = $scoreAdjusters;
     }
 
     public function score(BigFootSighting $sighting): BigFootSightingScore
