diff --git a/src/Scoring/ScoringFactorInterface.php b/src/Scoring/ScoringFactorInterface.php
index a99529c..0d66865 100644
--- a/src/Scoring/ScoringFactorInterface.php
+++ b/src/Scoring/ScoringFactorInterface.php
@@ -12,4 +12,6 @@ interface ScoringFactorInterface
      * This method should not throw an exception for any normal reason.
      */
     public function score(BigFootSighting $sighting): int;
+
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int;
 }
