diff --git a/src/Scoring/MaxScoreAdjuster.php b/src/Scoring/MaxScoreAdjuster.php
index 212fdf5..e531c56 100644
--- a/src/Scoring/MaxScoreAdjuster.php
+++ b/src/Scoring/MaxScoreAdjuster.php
@@ -10,4 +10,9 @@ class MaxScoreAdjuster implements ScoringFactorInterface
     {
         return min($finalScore, 100);
     }
+
+    public function score(BigFootSighting $sighting): int
+    {
+        return 0;
+    }
 }
