diff --git a/src/Scoring/CoordinatesFactor.php b/src/Scoring/CoordinatesFactor.php
index eb8b4fb..c80bbae 100644
--- a/src/Scoring/CoordinatesFactor.php
+++ b/src/Scoring/CoordinatesFactor.php
@@ -21,4 +21,9 @@ class CoordinatesFactor implements ScoringFactorInterface
 
         return $score;
     }
+
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int
+    {
+        return $finalScore;
+    }
 }
diff --git a/src/Scoring/DescriptionFactor.php b/src/Scoring/DescriptionFactor.php
index dc168ae..34e60c2 100644
--- a/src/Scoring/DescriptionFactor.php
+++ b/src/Scoring/DescriptionFactor.php
@@ -25,4 +25,9 @@ class DescriptionFactor implements ScoringFactorInterface
 
         return $score;
     }
+
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int
+    {
+        return $finalScore;
+    }
 }
diff --git a/src/Scoring/TitleFactor.php b/src/Scoring/TitleFactor.php
index ec1ac94..23fae2e 100644
--- a/src/Scoring/TitleFactor.php
+++ b/src/Scoring/TitleFactor.php
@@ -21,4 +21,9 @@ class TitleFactor implements ScoringFactorInterface
 
         return $score;
     }
+
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int
+    {
+        return $finalScore;
+    }
 }
