diff --git a/src/Scoring/MaxScoreAdjuster.php b/src/Scoring/MaxScoreAdjuster.php
index f818c81..789a355 100644
--- a/src/Scoring/MaxScoreAdjuster.php
+++ b/src/Scoring/MaxScoreAdjuster.php
@@ -4,15 +4,10 @@ namespace App\Scoring;
 
 use App\Entity\BigFootSighting;
 
-class MaxScoreAdjuster implements ScoringFactorInterface
+class MaxScoreAdjuster implements ScoreAdjusterInterface
 {
     public function adjustScore(int $finalScore, BigFootSighting $sighting): int
     {
         return $finalScore > 100 ? 100 : $finalScore;
     }
-
-    public function score(BigFootSighting $sighting): int
-    {
-        return 0;
-    }
 }
