diff --git a/src/Scoring/PhotoFactor.php b/src/Scoring/PhotoFactor.php
index d3ec048..a20c291 100644
--- a/src/Scoring/PhotoFactor.php
+++ b/src/Scoring/PhotoFactor.php
@@ -4,7 +4,7 @@ namespace App\Scoring;
 
 use App\Entity\BigFootSighting;
 
-class PhotoFactor implements ScoringFactorInterface
+class PhotoFactor implements ScoringFactorInterface, ScoreAdjusterInterface
 {
     public function score(BigFootSighting $sighting): int
     {
