diff --git a/src/Scoring/ScoringFactorInterface.php b/src/Scoring/ScoringFactorInterface.php
index a49a3e1..d4ca43d 100644
--- a/src/Scoring/ScoringFactorInterface.php
+++ b/src/Scoring/ScoringFactorInterface.php
@@ -6,6 +6,8 @@ use App\Entity\BigFootSighting;
 
 interface ScoringFactorInterface
 {
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int;
+
     /**
      * Return the score that should be added to the overall score.
      * This method should not throw an exception for any normal reason.
