diff --git a/src/Scoring/MaxScoreAdjuster.php b/src/Scoring/MaxScoreAdjuster.php
index 988db1d..13e1caf 100644
--- a/src/Scoring/MaxScoreAdjuster.php
+++ b/src/Scoring/MaxScoreAdjuster.php
@@ -8,4 +8,9 @@ class MaxScoreAdjuster implements ScoringFactorInterface
     {
         return $finalScore > 100 ? 100 : $finalScore;
     }
+
+    public function score(BigFootSighting $sighting): int
+    {
+        return 0;
+    }
 }
