diff --git a/src/Scoring/MaxScoreAdjuster.php b/src/Scoring/MaxScoreAdjuster.php
index 6cdaf8b..988db1d 100644
--- a/src/Scoring/MaxScoreAdjuster.php
+++ b/src/Scoring/MaxScoreAdjuster.php
@@ -4,4 +4,8 @@ namespace App\Scoring;
 
 class MaxScoreAdjuster implements ScoringFactorInterface
 {
+    public function adjustScore(int $finalScore, BigFootSighting $sighting): int
+    {
+        return $finalScore > 100 ? 100 : $finalScore;
+    }
 }
