diff --git a/src/Entity/BigFootSighting.php b/src/Entity/BigFootSighting.php
index 707a30f..bcf4ee1 100644
--- a/src/Entity/BigFootSighting.php
+++ b/src/Entity/BigFootSighting.php
@@ -60,6 +60,13 @@ class BigFootSighting
      */
     private $score = 0;
 
+    /**
+     * Stores an array of file paths
+     *
+     * @ORM\Column(type="json")
+     */
+    private $images = [];
+
     public function __construct()
     {
         $this->createdAt = new \DateTimeImmutable();
@@ -185,4 +192,14 @@ class BigFootSighting
 
         return $this;
     }
+
+    public function getImages(): array
+    {
+        return $this->images;
+    }
+
+    public function setImages(array $images)
+    {
+        $this->images = $images;
+    }
 }
