diff --git a/src/Model/DebuggableBigFootSightingScore.php b/src/Model/DebuggableBigFootSightingScore.php
index 1550d65..00e69d3 100644
--- a/src/Model/DebuggableBigFootSightingScore.php
+++ b/src/Model/DebuggableBigFootSightingScore.php
@@ -4,5 +4,17 @@ namespace App\Model;
 
 class DebuggableBigFootSightingScore extends BigFootSightingScore
 {
+    private float $calculationTime;
 
+    public function __construct(int $score, float $calculationTime)
+    {
+        parent::__construct($score);
+
+        $this->calculationTime = $calculationTime;
+    }
+
+    public function getCalculationTime(): float
+    {
+        return $this->calculationTime;
+    }
 }
