diff --git a/src/Entity/BigFootSighting.php b/src/Entity/BigFootSighting.php
index bcf4ee1..707a30f 100644
--- a/src/Entity/BigFootSighting.php
+++ b/src/Entity/BigFootSighting.php
@@ -60,13 +60,6 @@ class BigFootSighting
      */
     private $score = 0;
 
-    /**
-     * Stores an array of file paths
-     *
-     * @ORM\Column(type="json")
-     */
-    private $images = [];
-
     public function __construct()
     {
         $this->createdAt = new \DateTimeImmutable();
@@ -192,14 +185,4 @@ class BigFootSighting
 
         return $this;
     }
-
-    public function getImages(): array
-    {
-        return $this->images;
-    }
-
-    public function setImages(array $images)
-    {
-        $this->images = $images;
-    }
 }
