diff --git a/src/Service/SightingScorer.php b/src/Service/SightingScorer.php
index e027728..eba348c 100644
--- a/src/Service/SightingScorer.php
+++ b/src/Service/SightingScorer.php
@@ -3,6 +3,7 @@
 namespace App\Service;
 
 use App\Entity\BigFootSighting;
+use App\Scoring\PhotoFactor;
 use App\Model\BigFootSightingScore;
 use App\Scoring\ScoringFactorInterface;
 
@@ -22,6 +23,11 @@ class SightingScorer
     {
         $score = 0;
         foreach ($this->scoringFactors as $scoringFactor) {
+            // LSP violation and also OCP violation
+            if ($scoringFactor instanceof PhotoFactor && empty($sighting->getImages())) {
+                continue;
+            }
+
             $score += $scoringFactor->score($sighting);
         }
 
