diff --git a/src/Service/DebuggableSightingScorer.php b/src/Service/DebuggableSightingScorer.php
index 8edfd65..4670d43 100644
--- a/src/Service/DebuggableSightingScorer.php
+++ b/src/Service/DebuggableSightingScorer.php
@@ -4,11 +4,13 @@ namespace App\Service;
 
 use App\Entity\BigFootSighting;
 use App\Model\BigFootSightingScore;
+use App\Model\DebuggableBigFootSightingScore;
 
 class DebuggableSightingScorer extends SightingScorer
 {
-    public function score(BigFootSighting $sighting): BigFootSightingScore
+    public function score(BigFootSighting $sighting): DebuggableBigFootSightingScore
     {
-        return parent::score($sighting);
+        $bfsScore = parent::score($sighting);
+        return new DebuggableBigFootSightingScore($bfsScore->getScore(), 0);
     }
 }
