diff --git a/src/Form/BigfootSightingType.php b/src/Form/BigfootSightingType.php
index 3985a5f..d2948d3 100644
--- a/src/Form/BigfootSightingType.php
+++ b/src/Form/BigfootSightingType.php
@@ -5,6 +5,7 @@ namespace App\Form;
 use App\Entity\BigFootSighting;
 use Symfony\Component\Form\AbstractType;
 use Symfony\Component\Form\CallbackTransformer;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
 use Symfony\Component\Form\Extension\Core\Type\TextareaType;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\Component\OptionsResolver\OptionsResolver;
