diff --git a/src/Scoring/ScoringFactorInterface.php b/src/Scoring/ScoringFactorInterface.php
index 273c5a0..a49a3e1 100644
--- a/src/Scoring/ScoringFactorInterface.php
+++ b/src/Scoring/ScoringFactorInterface.php
@@ -6,5 +6,9 @@ use App\Entity\BigFootSighting;
 
 interface ScoringFactorInterface
 {
+    /**
+     * Return the score that should be added to the overall score.
+     * This method should not throw an exception for any normal reason.
+     */
     public function score(BigFootSighting $sighting): int;
 }
