diff --git a/src/Scoring/DescriptionFactor.php b/src/Scoring/DescriptionFactor.php
index 480d333..1ca9958 100644
--- a/src/Scoring/DescriptionFactor.php
+++ b/src/Scoring/DescriptionFactor.php
@@ -11,11 +11,11 @@ class DescriptionFactor implements ScoringFactorInterface
         $score = 0;
         $description = strtolower($sighting->getDescription());
 
-        if (strpos($description, 'hairy') !== false) {
+        if (stripos($description, 'hairy') !== false) {
             $score += 10;
         }
 
-        if (strpos($description, 'chased me') !== false) {
+        if (stripos($description, 'chased me') !== false) {
             $score += 20;
         }
 
diff --git a/src/Scoring/TitleFactor.php b/src/Scoring/TitleFactor.php
index 0d07ef0..ec1ac94 100644
--- a/src/Scoring/TitleFactor.php
+++ b/src/Scoring/TitleFactor.php
@@ -11,11 +11,11 @@ class TitleFactor implements ScoringFactorInterface
         $score = 0;
         $title = strtolower($sighting->getTitle());
 
-        if (strpos($title, 'hairy') !== false) {
+        if (stripos($title, 'hairy') !== false) {
             $score += 10;
         }
 
-        if (strpos($title, 'chased me') !== false) {
+        if (stripos($title, 'chased me') !== false) {
             $score += 20;
         }
 
diff --git a/src/Service/SightingScorer.php b/src/Service/SightingScorer.php
index b4efaff..b99dda5 100644
--- a/src/Service/SightingScorer.php
+++ b/src/Service/SightingScorer.php
@@ -4,9 +4,20 @@ namespace App\Service;
 
 use App\Entity\BigFootSighting;
 use App\Model\BigFootSightingScore;
+use App\Scoring\ScoringFactorInterface;
 
 class SightingScorer
 {
+    /**
+     * @var ScoringFactorInterface[]
+     */
+    private $scoringFactors;
+
+    public function __construct(array $scoringFactors)
+    {
+        $this->scoringFactors = $scoringFactors;
+    }
+
     public function score(BigFootSighting $sighting): BigFootSightingScore
     {
         $score = 0;
@@ -16,57 +27,4 @@ class SightingScorer
 
         return new BigFootSightingScore($score);
     }
-
-    private function evaluateCoordinates(BigFootSighting $sighting): int
-    {
-        $score = 0;
-        $lat = (float)$sighting->getLatitude();
-        $lng = (float)$sighting->getLongitude();
-
-        // California edge to edge coordinates
-        if (true
-            && $lat >= 32.5121 && $lat <= 42.0126
-            && $lng >= -114.1315 && $lat <= -124.6509
-        ) {
-            $score += 30;
-        }
-
-        return $score;
-    }
-
-    private function evaluateTitle(BigFootSighting $sighting): int
-    {
-        $score = 0;
-        $title = strtolower($sighting->getTitle());
-
-        if (stripos($title, 'hairy') !== false) {
-            $score += 10;
-        }
-
-        if (stripos($title, 'chased me') !== false) {
-            $score += 20;
-        }
-
-        return $score;
-    }
-
-    private function evaluateDescription(BigFootSighting $sighting): int
-    {
-        $score = 0;
-        $title = strtolower($sighting->getTitle());
-
-        if (stripos($title, 'hairy') !== false) {
-            $score += 10;
-        }
-
-        if (stripos($title, 'chased me') !== false) {
-            $score += 20;
-        }
-
-        if (stripos($title, 'using an Iphone') !== false) {
-            $score -= 50;
-        }
-
-        return $score;
-    }
 }
