diff --git a/src/Scoring/CoordinatesFactor.php b/src/Scoring/CoordinatesFactor.php
new file mode 100644
index 0000000..eb8b4fb
--- /dev/null
+++ b/src/Scoring/CoordinatesFactor.php
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Scoring;
+
+use App\Entity\BigFootSighting;
+
+class CoordinatesFactor implements ScoringFactorInterface
+{
+    public function score(BigFootSighting $sighting): int
+    {
+        $score = 0;
+        $lat = (float)$sighting->getLatitude();
+        $lng = (float)$sighting->getLongitude();
+
+        // California edge to edge coordinates
+        if ($lat >= 32.5121 && $lat <= 42.0126
+            && $lng >= -114.1315 && $lng <= -124.6509
+        ) {
+            $score += 30;
+        }
+
+        return $score;
+    }
+}
diff --git a/src/Scoring/DescriptionFactor.php b/src/Scoring/DescriptionFactor.php
new file mode 100644
index 0000000..9f7a133
--- /dev/null
+++ b/src/Scoring/DescriptionFactor.php
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Scoring;
+
+use App\Entity\BigFootSighting;
+
+class DescriptionFactor implements ScoringFactorInterface
+{
+    public function score(BigFootSighting $sighting): int
+    {
+        $score = 0;
+        $title = strtolower($sighting->getDescription());
+
+        if (stripos($title, 'hairy') !== false) {
+            $score += 10;
+        }
+
+        if (stripos($title, 'chased me') !== false) {
+            $score += 20;
+        }
+
+        if (stripos($title, 'using an iPhone') !== false) {
+            $score -= 50;
+        }
+
+        return $score;
+    }
+}
diff --git a/src/Scoring/TitleFactor.php b/src/Scoring/TitleFactor.php
new file mode 100644
index 0000000..ec1ac94
--- /dev/null
+++ b/src/Scoring/TitleFactor.php
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Scoring;
+
+use App\Entity\BigFootSighting;
+
+class TitleFactor implements ScoringFactorInterface
+{
+    public function score(BigFootSighting $sighting): int
+    {
+        $score = 0;
+        $title = strtolower($sighting->getTitle());
+
+        if (stripos($title, 'hairy') !== false) {
+            $score += 10;
+        }
+
+        if (stripos($title, 'chased me') !== false) {
+            $score += 20;
+        }
+
+        return $score;
+    }
+}
