diff --git a/src/Scoring/ScoringFactorInterface.php b/src/Scoring/ScoringFactorInterface.php
new file mode 100644
index 0000000..273c5a0
--- /dev/null
+++ b/src/Scoring/ScoringFactorInterface.php
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Scoring;
+
+use App\Entity\BigFootSighting;
+
+interface ScoringFactorInterface
+{
+    public function score(BigFootSighting $sighting): int;
+}
diff --git a/src/Service/SightingScorer.php b/src/Service/SightingScorer.php
index 812f191..b4efaff 100644
--- a/src/Service/SightingScorer.php
+++ b/src/Service/SightingScorer.php
@@ -39,11 +39,11 @@ class SightingScorer
         $score = 0;
         $title = strtolower($sighting->getTitle());
 
-        if (strpos($title, 'hairy') !== false) {
+        if (stripos($title, 'hairy') !== false) {
             $score += 10;
         }
 
-        if (strpos($title, 'chased me') !== false) {
+        if (stripos($title, 'chased me') !== false) {
             $score += 20;
         }
 
@@ -55,15 +55,15 @@ class SightingScorer
         $score = 0;
         $title = strtolower($sighting->getTitle());
 
-        if (strpos($title, 'hairy') !== false) {
+        if (stripos($title, 'hairy') !== false) {
             $score += 10;
         }
 
-        if (strpos($title, 'chased me') !== false) {
+        if (stripos($title, 'chased me') !== false) {
             $score += 20;
         }
 
-        if (strpos($title, 'using an Iphone') !== false) {
+        if (stripos($title, 'using an Iphone') !== false) {
             $score -= 50;
         }
 
