diff --git a/scenario.bkf b/scenario.bkf
index 972ffbd..fdbd745 100644
--- a/scenario.bkf
+++ b/scenario.bkf
@@ -2,3 +2,12 @@ name "Various scenarios for the site"
 
 # Can be override with option "--endpoint=http://newendpoint.com"
 endpoint "https://localhost:8000"
+
+scenario
+    name "Basic Visit"
+
+    visit url('/')
+        name "Homepage"
+
+    click link("Log In")
+        name "Log in page"
