diff --git a/scenario.bkf b/scenario.bkf
index 92be148..8a58fb0 100644
--- a/scenario.bkf
+++ b/scenario.bkf
@@ -10,6 +10,8 @@ scenario
         name "Homepage"
         expect status_code() == 200
         expect css("tbody.js-sightings-list tr").count() > 10
+        # won't work until we're using Blackfire environment
+        assert metrics.sql.queries.count < 30
 
     click link("Log In")
         name "Login page"
