diff --git a/src/EventSubscriber/BlackfireAutoProfileSubscriber.php b/src/EventSubscriber/BlackfireAutoProfileSubscriber.php
index 4b731a5..fb20634 100644
--- a/src/EventSubscriber/BlackfireAutoProfileSubscriber.php
+++ b/src/EventSubscriber/BlackfireAutoProfileSubscriber.php
@@ -41,7 +41,8 @@ class BlackfireAutoProfileSubscriber implements EventSubscriberInterface
     public static function getSubscribedEvents()
     {
         return [
-            RequestEvent::class => 'onRequestEvent',
+            // warning: adding a priority will run before routing & security
+            RequestEvent::class => ['onRequestEvent', 1000],
             TerminateEvent::class => 'onTerminateEvent',
         ];
     }
