diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index da05ffd..ea32e9e 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -21,11 +21,17 @@ class MainController extends AbstractController
      */
     public function homepage(BigFootSightingRepository $bigFootSightingRepository)
     {
+        $probe = \BlackfireProbe::getMainInstance();
+        $probe->enable();
         $sightings = $this->createSightingsPaginator(1, $bigFootSightingRepository);
 
-        return $this->render('main/homepage.html.twig', [
+        $response = $this->render('main/homepage.html.twig', [
             'sightings' => $sightings
         ]);
+
+        $probe->disable();
+
+        return $response;
     }
 
     /**
