diff --git a/src/Manager/UserManager.php b/src/Manager/UserManager.php
index 0bdc19a..0ed7c18 100644
--- a/src/Manager/UserManager.php
+++ b/src/Manager/UserManager.php
@@ -30,6 +30,13 @@ class UserManager
         $token = $this->createToken();
         $user->setConfirmationToken($token);
 
+        $user->setPassword(
+            $this->passwordEncoder->encodePassword($user, $plainPassword)
+        );
+
+        $this->entityManager->persist($user);
+        $this->entityManager->flush();
+
         $confirmationLink = $this->router->generate('check_confirmation_link', [
             'token' => $user->getConfirmationToken()
         ], UrlGeneratorInterface::ABSOLUTE_URL);
@@ -43,13 +50,6 @@ class UserManager
                 'confirmationLink' => $confirmationLink
             ]);
 
-        $user->setPassword(
-            $this->passwordEncoder->encodePassword($user, $plainPassword)
-        );
-
-        $this->entityManager->persist($user);
-        $this->entityManager->flush();
-
         $this->mailer->send($confirmationEmail);
 
         return $user;
diff --git a/src/Service/ConfirmationEmailSender.php b/src/Service/ConfirmationEmailSender.php
index dcb0e43..e3ca81a 100644
--- a/src/Service/ConfirmationEmailSender.php
+++ b/src/Service/ConfirmationEmailSender.php
@@ -2,7 +2,10 @@
 
 namespace App\Service;
 
+use App\Entity\User;
+use Symfony\Bridge\Twig\Mime\TemplatedEmail;
 use Symfony\Component\Mailer\MailerInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\RouterInterface;
 
 class ConfirmationEmailSender
@@ -15,4 +18,22 @@ class ConfirmationEmailSender
         $this->mailer = $mailer;
         $this->router = $router;
     }
+
+    public function send(User $user)
+    {
+        $confirmationLink = $this->router->generate('check_confirmation_link', [
+            'token' => $user->getConfirmationToken()
+        ], UrlGeneratorInterface::ABSOLUTE_URL);
+
+        $confirmationEmail = (new TemplatedEmail())
+            ->from('staff@example.com')
+            ->to($user->getEmail())
+            ->subject('Confirm your account')
+            ->htmlTemplate('emails/registration.html.twig')
+            ->context([
+                'confirmationLink' => $confirmationLink
+            ]);
+
+        $this->mailer->send($confirmationEmail);
+    }
 }
