diff --git a/src/Controllers/PictureController.php b/src/Controllers/PictureController.php
index 5c7c2c0..742efcf 100644
--- a/src/Controllers/PictureController.php
+++ b/src/Controllers/PictureController.php
@@ -26,17 +26,15 @@ class PictureController
         <p>All reported Sasquatch sightings, we will find the truth, upload your image to help us find Bigfoot!</p>
         <a class="btn btn-info btn-lg" href="upload">Upload your Bigfoot Image</a>
         <div class="my-5 row">
-        
+        {pictures}
+        </div>
 ';
+        $picturesHTML = '';
         foreach ( Picture::findAll() as $picture ) {
-            $contents .= $this->renderPicture( $picture );
+            $picturesHTML .= $this->renderPicture( $picture );
         }
 
-        $contents .= '
-        </div>
-       ';
-
-        return $this->render( $contents );
+        return $this->render( preg_replace( '/{pictures}/', $picturesHTML, $contents ) );
     }
 
     private function render( string $contents ) : string
