diff --git a/src/Controllers/PictureController.php b/src/Controllers/PictureController.php
index e557b06..c98a531 100644
--- a/src/Controllers/PictureController.php
+++ b/src/Controllers/PictureController.php
@@ -80,20 +80,10 @@ class PictureController
                         $_POST['location']
                     );
 
-                    return $this->render(
-                        '
-            <h2>Sasquatch Spotted! </h2>
-            <p>Click <u><a class="text-white" href="/">here</a></u> to see your image and the rest of the sightings!</p>'
-                    );
+                    return $this->render( $this->getSuccessfullUploadResponseHTML() );
                 } catch ( \Exception $e ) {
 
-                    return $this->render(
-                        '
-            <h2 class="mb-4 text-center">Sneaky like a Sasquatch, that upload didn\'t work!</h2>
-            <p class="text-center"><a class="btn btn-light btn-lg" href="upload">Try Again</a></p>
-            
-            '
-                    );
+                    return $this->render( $this->getFailedUploadResponseHTML() );
                 }
             }
         } else {
@@ -181,4 +171,26 @@ class PictureController
 ';
         return $contents;
     }
+
+    /**
+     * @return string
+     */
+    private function getSuccessfullUploadResponseHTML(): string
+    {
+        return '
+            <h2>Sasquatch Spotted! </h2>
+            <p>Click <u><a class="text-white" href="/">here</a></u> to see your image and the rest of the sightings!</p>';
+    }
+
+    /**
+     * @return string
+     */
+    private function getFailedUploadResponseHTML(): string
+    {
+        return '
+            <h2 class="mb-4 text-center">Sneaky like a Sasquatch, that upload didn\'t work!</h2>
+            <p class="text-center"><a class="btn btn-light btn-lg" href="upload">Try Again</a></p>
+            
+            ';
+    }
 }
\ No newline at end of file
