diff --git a/src/Manager/UserManager.php b/src/Manager/UserManager.php
index dbcc3a7..57a2ad0 100644
--- a/src/Manager/UserManager.php
+++ b/src/Manager/UserManager.php
@@ -4,6 +4,7 @@ namespace App\Manager;
 
 use App\Entity\User;
 use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Mailer\MailerInterface;
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
 use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
@@ -13,12 +14,14 @@ class UserManager
     private $passwordEncoder;
     private $entityManager;
     private $router;
+    private $mailer;
 
-    public function __construct(UserPasswordEncoderInterface $passwordEncoder, EntityManagerInterface $entityManager, RouterInterface $router)
+    public function __construct(UserPasswordEncoderInterface $passwordEncoder, EntityManagerInterface $entityManager, RouterInterface $router, MailerInterface $mailer)
     {
         $this->passwordEncoder = $passwordEncoder;
         $this->entityManager = $entityManager;
         $this->router = $router;
+        $this->mailer = $mailer;
     }
 
     public function register(User $user, string $plainPassword)
