diff --git a/src/Controllers/PictureController.php b/src/Controllers/PictureController.php
index 2c699c4..e557b06 100644
--- a/src/Controllers/PictureController.php
+++ b/src/Controllers/PictureController.php
@@ -73,13 +73,28 @@ class PictureController
                 return $this->error400();
             } else {
 
-                return $this->render(
+                try {
                     $this->storeUploadedPicture(
                         $_FILES['newPicture'],
                         $_POST['author'],
                         $_POST['location']
-                    )
-                );
+                    );
+
+                    return $this->render(
+                        '
+            <h2>Sasquatch Spotted! </h2>
+            <p>Click <u><a class="text-white" href="/">here</a></u> to see your image and the rest of the sightings!</p>'
+                    );
+                } catch ( \Exception $e ) {
+
+                    return $this->render(
+                        '
+            <h2 class="mb-4 text-center">Sneaky like a Sasquatch, that upload didn\'t work!</h2>
+            <p class="text-center"><a class="btn btn-light btn-lg" href="upload">Try Again</a></p>
+            
+            '
+                    );
+                }
             }
         } else {
 
@@ -87,7 +102,14 @@ class PictureController
         }
     }
 
-    private function storeUploadedPicture(array $uploadedFile, string $author, string $location ) : string
+    /**
+     * @param array $uploadedFile
+     * @param string $author
+     * @param string $location
+     * @return string
+     * @throws \Exception
+     */
+    private function storeUploadedPicture(array $uploadedFile, string $author, string $location )
     {
         $destination = __DIR__ . '/../../uploads/' . basename($uploadedFile['name']);
         if ( move_uploaded_file( $uploadedFile['tmp_name'], $destination ) ) {
@@ -100,17 +122,9 @@ class PictureController
             ];
 
             file_put_contents( $metaDataFileName, json_encode( $metaData ) );
-
-            return '
-            <h2>Sasquatch Spotted! </h2>
-            <p>Click <u><a class="text-white" href="/">here</a></u> to see your image and the rest of the sightings!</p>';
         } else {
 
-            return '
-            <h2 class="mb-4 text-center">Sneaky like a Sasquatch, that upload didn\'t work!</h2>
-            <p class="text-center"><a class="btn btn-light btn-lg" href="upload">Try Again</a></p>
-            
-            ';
+            throw new \Exception( basename($uploadedFile['name'] ).' couldn\'t be moved to '. $destination );
         }
     }
 
