diff --git a/src/Manager/UserManager.php b/src/Manager/UserManager.php
index 5a3f912..dbcc3a7 100644
--- a/src/Manager/UserManager.php
+++ b/src/Manager/UserManager.php
@@ -4,24 +4,32 @@ namespace App\Manager;
 
 use App\Entity\User;
 use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Routing\RouterInterface;
 use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
 
 class UserManager
 {
     private $passwordEncoder;
     private $entityManager;
+    private $router;
 
-    public function __construct(UserPasswordEncoderInterface $passwordEncoder, EntityManagerInterface $entityManager)
+    public function __construct(UserPasswordEncoderInterface $passwordEncoder, EntityManagerInterface $entityManager, RouterInterface $router)
     {
         $this->passwordEncoder = $passwordEncoder;
         $this->entityManager = $entityManager;
+        $this->router = $router;
     }
 
     public function register(User $user, string $plainPassword)
     {
         $token = $this->createToken();
         $user->setConfirmationToken($token);
-        
+
+        $confirmationLink = $this->router->generate('check_confirmation_link', [
+            'token' => $user->getConfirmationToken()
+        ], UrlGeneratorInterface::ABSOLUTE_URL);
+
         $user->setPassword(
             $this->passwordEncoder->encodePassword($user, $plainPassword)
         );
