diff --git a/src/Controllers/PictureController.php b/src/Controllers/PictureController.php
index 7618934..5c7c2c0 100644
--- a/src/Controllers/PictureController.php
+++ b/src/Controllers/PictureController.php
@@ -6,6 +6,18 @@ use sasquatch\Models\Picture;
 
 class PictureController
 {
+    private function renderPicture( Picture $picture ) : string
+    {
+        return "
+            <div class='col-sm-4 mb-5'>
+                <img src='show?file={$picture->getFileName()}' class='big-foot-img'/> 
+                <p class='mt-3 mb-0'>Image taken by: <strong>{$picture->getAuthor()}</strong></p> 
+                <p class='mb-0'>Coordinates: <strong>{$picture->getLocation()}</strong></p>
+                <p class='mb-0'>Date: <strong>{$picture->getDate()->format('d/m/Y')}</strong></p>
+            </div>
+        ";
+    }
+
     public function index() : string
     {
         $contents = '
@@ -17,14 +29,7 @@ class PictureController
         
 ';
         foreach ( Picture::findAll() as $picture ) {
-            $contents .= "
-            <div class='col-sm-4 mb-5'>
-                <img src='show?file={$picture->getFileName()}' class='big-foot-img'/>
-                <p class='mt-3 mb-0'>Image taken by: <strong>{$picture->getAuthor()}</strong></p> 
-                <p class='mb-0'>Coordinates: <strong>{$picture->getLocation()}</strong></p>
-                <p class='mb-0'>Date: <strong>{$picture->getDate()->format('d/m/Y')}</strong></p>
-            </div>
-        ";
+            $contents .= $this->renderPicture( $picture );
         }
 
         $contents .= '
