diff --git a/src/Controller/RegistrationController.php b/src/Controller/RegistrationController.php
index 0ccf9c4..291d693 100644
--- a/src/Controller/RegistrationController.php
+++ b/src/Controller/RegistrationController.php
@@ -42,7 +42,7 @@ class RegistrationController extends AbstractController
     }
 
     /**
-     * @Route("/confirm/{token}", name="check_confirmation_link")
+     * @Route("/confirm/{token}", name="check_confirmation_link", methods={"GET"})
      */
     public function confirmAction(string $token, UserRepository $userRepository, EntityManagerInterface $entityManager)
     {
