diff --git a/tests/Controller/MainControllerTest.php b/tests/Controller/MainControllerTest.php
index 9c93001..b4c5427 100644
--- a/tests/Controller/MainControllerTest.php
+++ b/tests/Controller/MainControllerTest.php
@@ -14,16 +14,11 @@ class MainControllerTest extends WebTestCase
     {
         $client = static::createClient();
 
-        $blackfireConfig = (new Configuration())
-            ->assert('metrics.http.requests.count == 1');
-
-        $this->assertBlackfire($blackfireConfig, function() use ($client) {
-            $client->request('GET', '/api/github-organization');
+        $client->request('GET', '/api/github-organization');
 
-            $this->assertResponseIsSuccessful();
-            $data = json_decode($client->getResponse()->getContent(), true);
-            $this->assertArrayHasKey('organization', $data);
-        });
+        $this->assertResponseIsSuccessful();
+        $data = json_decode($client->getResponse()->getContent(), true);
+        $this->assertArrayHasKey('organization', $data);
     }
 
     public function testGetGitHubOrganizationBlackfireHttpRequests()
