diff --git a/src/Controller/MainController.php b/src/Controller/MainController.php
index da05ffd..628bc35 100644
--- a/src/Controller/MainController.php
+++ b/src/Controller/MainController.php
@@ -55,8 +55,8 @@ class MainController extends AbstractController
     public function gitHubOrganizationInfo(GitHubApiHelper $apiHelper)
     {
         $organizationName = 'SymfonyCasts';
-        $organization = $apiHelper->getOrganizationInfo($organizationName);
         $repositories = $apiHelper->getOrganizationRepositories($organizationName);
+        $organization = $apiHelper->getOrganizationInfo($organizationName);
 
         return $this->json([
             'organization' => $organization,
diff --git a/src/GitHub/GitHubApiHelper.php b/src/GitHub/GitHubApiHelper.php
index f33c217..9acc60f 100644
--- a/src/GitHub/GitHubApiHelper.php
+++ b/src/GitHub/GitHubApiHelper.php
@@ -8,6 +8,8 @@ class GitHubApiHelper
 {
     private $httpClient;
 
+    private $githubOrganizations = [];
+
     public function __construct(HttpClientInterface $httpClient)
     {
         $this->httpClient = $httpClient;
@@ -15,6 +17,11 @@ class GitHubApiHelper
 
     public function getOrganizationInfo(string $organization): GitHubOrganization
     {
+        // optimization in case getOrganizationRepositories is called first
+        if (isset($this->githubOrganizations[$organization])) {
+            return $this->githubOrganizations[$organization];
+        }
+
         $response = $this->httpClient->request('GET', 'https://api.github.com/orgs/'.$organization);
 
         $data = $response->toArray();
@@ -35,12 +42,24 @@ class GitHubApiHelper
        $data = $response->toArray();
 
        $repositories = [];
+       $publicRepoCount = 0;
        foreach ($data as $repoData) {
            $repositories[] = new GitHubRepository(
                $repoData['name'],
                $repoData['html_url'],
                \DateTimeImmutable::createFromFormat('Y-m-d\TH:i:s\Z', $repoData['updated_at'])
            );
+
+           if (false === $repoData['private']) {
+               ++$publicRepoCount;
+           }
+       }
+
+       if (!isset($this->githubOrganizations[$organization])) {
+           $this->githubOrganizations[$organization] = new GitHubOrganization(
+               $data[0]['owner']['login'],
+               $publicRepoCount
+           );
        }
 
        return $repositories;
