diff --git a/tests/Controller/MainControllerTest.php b/tests/Controller/MainControllerTest.php
index ca0db9f..2108ad8 100644
--- a/tests/Controller/MainControllerTest.php
+++ b/tests/Controller/MainControllerTest.php
@@ -2,17 +2,27 @@
 
 namespace App\Tests\Controller;
 
+use Blackfire\Bridge\PhpUnit\TestCaseTrait;
+use Blackfire\Profile\Configuration;
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
 
 class MainControllerTest extends WebTestCase
 {
+    use TestCaseTrait;
+
     public function testGetGitHubOrganization()
     {
         $client = static::createClient();
 
-        $client->request('GET', '/api/github-organization');
-        $this->assertResponseIsSuccessful();
-        $data = json_decode($client->getResponse()->getContent(), true);
-        $this->assertArrayHasKey('organization', $data);
+        $blackfireConfig = (new Configuration())
+            ->assert('metrics.http.requests.count == 1');
+
+        $this->assertBlackfire($blackfireConfig, function() use ($client) {
+            $client->request('GET', '/api/github-organization');
+
+            $this->assertResponseIsSuccessful();
+            $data = json_decode($client->getResponse()->getContent(), true);
+            $this->assertArrayHasKey('organization', $data);
+        });
     }
 }
