diff --git a/src/EventSubscriber/AgreeToTermsSubscriber.php b/src/EventSubscriber/AgreeToTermsSubscriber.php
index d634d4d..227a6a8 100644
--- a/src/EventSubscriber/AgreeToTermsSubscriber.php
+++ b/src/EventSubscriber/AgreeToTermsSubscriber.php
@@ -43,6 +43,11 @@ class AgreeToTermsSubscriber implements EventSubscriberInterface
         //$latestTermsDate = new \DateTimeImmutable('2019-10-15');
         $latestTermsDate = new \DateTimeImmutable('-1 year');
 
+        // user is up-to-date!
+        if ($user->getAgreedToTermsAt() >= $latestTermsDate) {
+            return;
+        }
+
         $form = $this->formFactory->create(AgreeToUpdatedTermsFormType::class);
 
         $html = $this->twig->render('main/agreeUpdatedTerms.html.twig', [
@@ -54,11 +59,6 @@ class AgreeToTermsSubscriber implements EventSubscriberInterface
         // we know the user doesn't need to see the form!
         $this->entrypointLookup->reset();
 
-        // user is up-to-date!
-        if ($user->getAgreedToTermsAt() >= $latestTermsDate) {
-            return;
-        }
-
         $response = new Response($html);
         $event->setResponse($response);
     }
